app/client/cli/servicer.go

```diff
@@ -115,13 +119,43 @@ func validateServicer(session *rpc.Session, servicerAddress string) (*rpc.Protoc
 	return nil, fmt.Errorf("Error getting servicer: address %s does not match any servicers in the session %d", servicerAddress, session.SessionNumber)
 }
 
+// ADDTEST: add either unit tests or integration tests using kvstore
```

Comment:
I feel like we should add one in this PR, seems small and simple?

Reply:
Thank you, fixed.